tests/cpp/test_matmul_aten_evaluation.cpp

```diff
@@ -334,4 +334,83 @@ TEST_F(MatmulATenEvaluationTest, MatmulBiasAlpha) {
 
   EXPECT_TRUE(at::allclose(out[0], out_ref));
 }
+
+TEST_F(MatmulATenEvaluationTest, Linear) {
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+
+  int64_t m = 32, n = 64, k = 128;
+  std::vector<int64_t> a_shape{m, k}, b_shape{n, k}, out_shape{m, n};
+
+  auto tv0 = makeConcreteTensor(a_shape, DataType::Half);
+  auto tv1 = makeConcreteTensor(b_shape, DataType::Half);
+  auto tv0b = broadcast(tv0, {false, true, false}); // [M, 1, K]
+  auto tv1b = broadcast(tv1, {true, false, false}); // [1, N, K]
```

Comment:
If this is how we plan to implement linear op, then could you pattern match the location of the K axis in order to determine whether its matmul or linear? Since matmul will have K in the middle position instead.

Reply:
Using `getMmaLayout` will be the same, right?